internal/venice-common/src/main/java/com/linkedin/venice/pubsub/adapter/kafka/ApacheKafkaUtils.java

```diff
@@ -0,0 +1,15 @@
+package com.linkedin.venice.pubsub.adapter.kafka;
+
+import com.linkedin.venice.pubsub.api.PubsubMessageHeaders;
+import org.apache.kafka.common.header.internals.RecordHeaders;
+
+
+public class ApacheKafkaUtils {
+  public static RecordHeaders convertToKafkaSpecificHeaders(PubsubMessageHeaders headers) {
+    RecordHeaders recordHeaders = new RecordHeaders();
+    if (headers != null) {
+      headers.toList().forEach(header -> recordHeaders.add(header.key(), header.value()));
+    }
+    return recordHeaders;
+  }
```

Comment:
When passing null `headers`, let's directly return a static final `EMPTY_HEADER` instantiated as a property of this class, rather than instantiating a `new RecordHeaders()` every time.